netmap/migration_test.go

```diff
@@ -24,17 +25,45 @@ func TestMigration(t *testing.T) {
 	require.NoError(t, err)
 }
 
-var notaryDisabledKey = []byte("notary")
+var (
+	notaryDisabledKey = []byte("notary")
+
+	newEpochSubsNewPrefix = []byte("newEpochSubscribers")
+	containerHashOldKey   = []byte("containerScriptHash")
+	balanceHashOldKey     = []byte("balanceScriptHash")
+)
 
 func testMigrationFromDump(t *testing.T, d *dump.Reader) {
+	var containerHash util.Uint160
+	var balanceHash util.Uint160
+	var err error
+
 	// init test contract shell
-	c := migration.NewContract(t, d, "netmap", migration.ContractOptions{})
+	c := migration.NewContract(t, d, "netmap", migration.ContractOptions{
+		StorageDumpHandler: func(key, value []byte) {
+			if bytes.Equal(containerHashOldKey, key) {
+				containerHash, err = util.Uint160DecodeBytesLE(value)
+				require.NoError(t, err)
+
+				return
+			}
+
+			if bytes.Equal(balanceHashOldKey, key) {
+				balanceHash, err = util.Uint160DecodeBytesLE(value)
+				require.NoError(t, err)
+
+				return
+			}
+		},
+	})
+
+	if containerHash.Equals(util.Uint160{}) || balanceHash.Equals(util.Uint160{}) {
```

Comment:
```suggestion
	require.NotZerof(t, containerHash, "missing storage item %q with the Container contract address", containerHashOldKey)
	require.NotZerof(t, balanceHash, "missing storage item %q with the Balance contract address", balanceHashOldKey)
```

Reply:
not expect it to fail ever (and if fail, it wont be a problem to understand the reason) but fixed